app-ios/Sources/SearchFeature/SelectButtonView.swift

```diff
@@ -0,0 +1,57 @@
+import Assets
+import Model
+import SwiftUI
+import Theme
+
+struct SelectButtonView: View {
+    let title: String
+    let selected: Bool
+    let onSelect: @Sendable () -> Void
+    let onDeselect: @Sendable () -> Void
+    var body: some View {
+        HStack(spacing: 20) {
+            Button {
+                selected ? onDeselect() : onSelect()
+            } label: {
+                Group {
+                    selected ?
+                        Image(systemName: "checkmark.square.fill")
+                            .resizable()
+                            .frame(width: 18, height: 18)
```

Comment:
`.frame` should be removed and add this to Group's modifier

Reply:
Fixed in [711fd31](https://github.com/DroidKaigi/conference-app-2022/pull/815/commits/711fd313b387085e37774f9d64d5bda062f5f9ac)